manager/templates/default/dashboard/rssitem.tpl

```diff
@@ -1,5 +1,5 @@
 <div class="news_article">
     <h2><a href="[[+link]]" target="_blank">[[+title]]</a></h2>
-    <span class="content">[[+description]]</span>
-    <span class="date_stamp">[[+pubdate]]</span>
+    <div class="content">[[+description]]</div>
+    <div class="date_stamp">[[+pubdate:strtotime:date=`%Y-%m-%d`]]</div>
```

Comment:
There is a system setting `manager_date_format` for customizing date formats in the manager. The format is set without `%`, we need to look how we can apply this setting here.